api/client/cli.go

```diff
@@ -93,10 +93,11 @@ func (cli *DockerCli) Subcmd(name, signature, description string, exitOnError bo
 	flags := flag.NewFlagSet(name, errorHandling)
 	flags.Usage = func() {
 		options := ""
+		tempSignature := " " + signature
```

Comment:
OK, I got what u mean and so agreed with u.
I will just test it and then commit if all command's 'help usage' work well in that way.
thanks very much